js.go

```diff
@@ -181,9 +185,28 @@ func (nc *Conn) JetStream(opts ...JSOpt) (JetStreamContext, error) {
 			return nil, err
 		}
 	}
+	return js, nil
+}
+
+// BindJetStream returns a JetStreamContext for messaging and stream management that will NOT create
+// underlying objects like stream or durable. It assumes they exist already.
+func (nc *Conn) BindJetStream(opts ...JSOpt) (JetStreamContext, error) {
```

Comment:
I think just JetStream interface might work better since JetStreamManagement requires being able to use the API.
```suggestion
func (nc *Conn) BindJetStream(opts ...JSOpt) (JetStream, error) {
```

Reply:
I would be ok if you can do that as these calls to create streams/durables are explicit.

Reply:
ok sounds good